docs/template.html

```diff
@@ -441,6 +442,9 @@
       <!--CONTENT-->
     </article>
   </div>
+  <script>
```

Comment:
As I mentioned before, this specific for a template that has search with the search index embedded in the HTML, I'd appreciate if you published a separate template that we could link to from the README